modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/IS/oauth/token_renewal_per_request_enabled.toml

```diff
@@ -0,0 +1,35 @@
+[server]
+pattern = "DEV"  #possible values, DEV,HA,HA_WITH_ANALYTICS
+hostname = "localhost"
+node_ip = "127.0.0.1"
+
+[super_admin]
+username = "admin"
+password = "admin"
+create_admin_account = true
+
+[user_store]
+type = "read_write_ldap"
+connection_url = "ldap://localhost:${Ports.EmbeddedLDAP.LDAPServerPort}"
+connection_name = "uid=admin,ou=system"
+connection_password = "admin"
+base_dn = "dc=wso2,dc=org"      #refers the base dn on which the user and group search bases will be generated
+
+[keystore.tls]
+name = "wso2carbon.jks"
+password = "wso2carbon"
+
+[oauth]
+prompt_consent = false
+
+[oauth.token_renewal]
+renew_access_token_per_request = true
+
+[authentication.consent]
+prompt = false
+
+[event.default_listener.consent_mgt_handler]
+enable = false
+
+
+
```

Comment:
Remove additional new lines

Reply:
Fixed - https://github.com/wso2/product-is/pull/7059